docs/gettingstarted/index.md

```diff
@@ -48,6 +48,16 @@ These instructions assume that you have created the project folder as described
 
 If all the steps worked correctly, you should now be able to choose the Minecraft run tasks from the dropdown, and then click the Run/Debug buttons to test your setup.
 
+Using Forge with NetBeans 8.*
+-----------------------------
+1. Open terminal inside the folder where you extracted Forge MDK/SRC;
+2. Run `gradlew setupDecompWorkspace --refresh-dependencies` and wait for it to finish;
+3. Make sure gradle's task finished properly. If not - consult it's output to fix erors (common issues are firewall and the `JAVA_HOME` environment variable not pointing to a valid JDK);
```

Comment:
* `If not -` should be `If not,`
* `it's output` should be `its output`
* `erors` should be `errors`